cmd/nodecmd/create.go

```diff
@@ -78,21 +78,26 @@ func getNewKeyPairName(ec2Svc *ec2.EC2) (string, error) {
 
 // createClusterNodeConfig creates node config and save it in .avalanche-cli/nodes/{instanceID}
 // also creates cluster config in .avalanche-cli/nodes storing various key pair and security group info for all clusters
-func createClusterNodeConfig(nodeID, region, ami, keyPairName, certPath, sg, eip, clusterName string) error {
-	nodeConfig := models.NodeConfig{
-		NodeID:        nodeID,
-		Region:        region,
-		AMI:           ami,
-		KeyPair:       keyPairName,
-		CertPath:      certPath,
-		SecurityGroup: sg,
-		ElasticIP:     eip,
-	}
-	err := app.CreateNodeCloudConfigFile(nodeID, &nodeConfig)
-	if err != nil {
-		return err
+func createClusterNodeConfig(nodeIDs, publicIPs []string, region, ami, keyPairName, certPath, sg, clusterName string) error {
+	for i := range nodeIDs {
+		nodeConfig := models.NodeConfig{
```

Comment:
So I guess one caveat in our docs should be we can only create multiple nodes when they have the same config. Multiple configs requires running the command multiple times.